src/components/Calculator.js

```diff
@@ -2,7 +2,11 @@ import React, { useState } from 'react';
 import calculate from '../logic/calculator';
 
 function CalculatorDesign() {
-  const [inputs, setInput] = useState({ total: 0, next: null, operation: null });
+  const [inputs, setInput] = useState({
+    total: 0,
+    next: null,
+    operation: null,
```

Comment:
- [ ] Please fix your calculator, I can't do any operation. please check your **operation.js** and fix your logic.